compose-common.yml

```diff
@@ -12,35 +12,49 @@ base:
     # --link some-rabbit:rabbit "just works"
     - DJANGO_CELERY_BROKER_URL=amqp://guest@rabbit
     - DJANGO_DATABASE_URL=postgis://postgres@database/postgres
-    - DJANGO_EMAIL_HOST=localhost
-    # Settings above likely need to be changed, depending on  the environment
+    - APP_PORT=8000
+    - APP_HOST=0.0.0.0
+
+    # Logging
+    - PYTHONUNBUFFERED=non-empty-string
```

Comment:
You have to set this to a non-empty-string, self documenting, I thought, and I thought wrong. ;-)